apollo-router/tests/integration/subscriptions/ws_passthrough.rs

```diff
@@ -653,16 +643,7 @@ async fn test_subscription_ws_passthrough_on_config_reload() -> Result<(), BoxEr
     assert_eq!(total_active, 1);
     assert_eq!(total_active + total_terminating, 1);
 
-    match verify_subscription_events(stream, expected_events, true)
-        .await
-        .map_err(|e| format!("Event verification failed: {}", e))
-    {
-        Ok(_events) => {}
-        Err(err) if err.contains("unexpected EOF") => {
```

Comment:
@carodewig I wasn't able to produce these errors. I think ideally they wouldn't occur, but I assume you put this in because they did 😅  Do you still see them when you run tests?